source/Calamari/Commands/KubernetesObjectStatusReporterCommand.cs

```diff
@@ -70,10 +74,14 @@ protected override void Execute(KubernetesObjectStatusReporterCommandInput input
             }
         }
 
-        private void ConfigureKubectl()
+        private void ConfigureKubectl(string workingDirectory)
         {
             var kubeConfig = variables.Get(SpecialVariables.KubeConfig);
-            var environmentVars = new Dictionary<string, string> {{"KUBECONFIG", kubeConfig}};
+            var environmentVars = new Dictionary<string, string>
+            {
+                ["KUBECONFIG"] = kubeConfig,
+                ["AZURE_CONFIG_DIR"] = Path.Combine(workingDirectory, "azure-cli")
```

Comment:
I wonder if we should share this part of the code so they never get out of sync?